app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -570,11 +570,15 @@ private fun ConversationScreen(
             }
         },
         snackbarHost = {
-            SwipeDismissSnackbarHost(
+            SnackbarHost(
```

Comment:
Shouldn't we take the chance and update the Scaffold here in Conversation screen to `WireScaffold`? Otherwise if any bug happens again with this snackbar library, and we rewrite the class, we will need to remember updating it here too.

Reply:
The big change was about to not have here WireScaffold because padding behaviour works different here